src/lib/database/schemas/Version1Schema.ts

```diff
@@ -42,34 +42,36 @@ const DbVideoZod = z.object({
 })
 
 
-const CommonDefinition = {
```

Comment:
why did you get rid of the Common Definition?

Reply:
Realized this after I had already done it: I got the idea that since the naming system will be different between tricks and combos from here on, I'll have to get rid of the common definition. I later realized that I simply could have moved out the namings and kept the common definitions. But at that point I asked myself whether it would make sense to go back and came to the conclusion that since the two things (tricks and combos) are somewhat but not strongly linked semantically having the common definition primarily would serve the purpose of cutting down the LOC. Since it wouldn't save that many lines, I decided against it.
But if you have a good reason why it would be better to keep the common definition, please let me know, I don't feel strongly about this decision and am happy to revert back.

Reply:
I'd prefer to keep the "common" part in one definition — simply so that we don't get into a situation where we do a change on the common part, but forget to at it to one of the schemas.

Reply:
Sure, I'll revert it back